user_docs/en/userGuide.md

```diff
@@ -2163,6 +2163,15 @@ This means that you do not have to scroll the display at the end of each line ev
 This may allow for more fluent reading of large amounts of text.
 It is disabled by default.
 
+##### Show paragraphs start {#BrailleShowParagraphStart}
```

Comment:
I think the plural of the word "paragraph" is not correct usage here. I'm not 100% sure of that, but reading it in English it just "doesn't sound right".

```suggestion
##### Show paragraph start {#BrailleShowParagraphStart}
```